### PR TITLE
feature: README updates to more clearly list out dependencies and how to install

### PR DESCRIPTION
**Pull request by sandragolden**
### What does this PR do?
README updates only, no code. Updated dependency list to include all current dependencies (I hope). We should probably get this updated in our developer docs as well, for example [here](https://developer.salesforce.com/docs/atlas.en-us.238.0.b2b_b2c_comm_dev.meta/b2b_b2c_comm_dev/b2b_b2c_comm_install_plugins.htm).

### What issues does this PR fix or reference?
no issue, just adding dependencies to try and help customers use the tool without errors or a lot of troubleshooting, as is witnessed in issue #64

### Functionality Before
****
Before Update 1:
****
[commerce-on-lightning-orchestration](https://github.com/forcedotcom/commerce-on-lightning-orchestration) - commerce-orchestration

-   scratchorgs

[commerce-on-lightning](https://github.com/forcedotcom/commerce-on-lightning) - @salesforce/commerce <---- THIS PLUGIN

-   store
****
Before Update 2:
****
#### Install

```bash
sfdx plugins:install @salesforce/commerce
```

### Functionality After

****
After Update 1:
****
-   [commerce-on-lightning](https://github.com/forcedotcom/commerce-on-lightning) - @salesforce/commerce <---- THIS PLUGIN
    -   create stores
    -   install command: `sfdx plugins:install @salesforce/commerce`
    -   install specific version:`sfdx plugins:install @salesforce/commerce@242.0.26`

#### Plugin Dependencies

1. [commerce-on-lightning-orchestration](https://github.com/forcedotcom/commerce-on-lightning-orchestration) - commerce-orchestration
    - scratchorgs
    - install command: `sfdx plugins:install commerce-orchestration`
2. [sfdx-1commerce-plugin](https://github.com/forcedotcom/sfdx-1commerce-plugin) - 1commerce
    - API calls for product import and search index
    - install command: `sfdx plugins:install 1commerce`
3. [shane-sfdx-plugins](https://github.com/mshanemc/shane-sfdx-plugins) - shane-sfdx-plugins
    - public plugin for imports
    - install command: `sfdx plugins:install shane-sfdx-plugin

****
After Update 2:
****
#### Install

```bash
sfdx plugins:install @salesforce/commerce

# install dependencies
sfdx plugins:install commerce-orchestration
sfdx plugins:install 1commerce
sfdx plugins:install shane-sfdx-plugins
```

### How to Test/Testing Effort 
n/a

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,13 +18,22 @@ This plugin is used to create and setup either a B2B or a B2C store with data wi
...
-
--   store
+-   [commerce-on-lightning](https://github.com/forcedotcom/commerce-on-lightning) - @salesforce/commerce <---- THIS PLUGIN
+    -   create stores
+    -   install command: `sfdx plugins:install @salesforce/commerce`
+    -   install specific version:`sfdx plugins:install @salesforce/commerce@242.0.26`
+
+#### Plugin Dependencies
+
+1. [commerce-on-lightning-orchestration](https://github.com/forcedotcom/commerce-on-lightning-orchestration) - commerce-orchestration
```
Comment (unandyala) [inline README.md:28]: this plugin is no longer a dependency. The features have been added to this (commerce-on-lightning) plugin

File: README.md [hunk 2]
```diff
@@ -18,13 +18,22 @@ This plugin is used to create and setup either a B2B or a B2C store with data wi
...
+    - scratchorgs
+    - install command: `sfdx plugins:install commerce-orchestration`
+2. [sfdx-1commerce-plugin](https://github.com/forcedotcom/sfdx-1commerce-plugin) - 1commerce
```
Comment (unandyala) [inline README.md:31]: this plugin is no longer a dependency. The features have been added to this (commerce-on-lightning) plugin

File: README.md (hunk 1 above)
Comment (sandragolden) [reply README.md]: thanks! removed as dependency

File: README.md (hunk 2 above)
Comment (sandragolden) [reply README.md]: thanks! removed as dependency

**Review by tarcang — DISMISSED** [review]

**Review by unandyala — DISMISSED** [review]

**Review by unandyala — APPROVED** [review]

**Review by bredmond-sf — APPROVED** [review]

**Pull request merged**